backend/src/main/scala/cromwell/backend/model/MemorySize.scala

```diff
@@ -1,4 +1,4 @@
-package cromwell.backend.validation
+package cromwell.backend.model
```

Comment:
I would rather keep things with their logical topics and not in one grab bag of stuff